### DISCUSSION TITLE
application is not listed in play store in Arabic but works in English

### DISCUSSION THREAD

**Discussion by muhammedelsepa3y** [community/community#118396] [Programming Help]
### Body

I have published an application in Play store, and in Store Listing (in dev console) I have added Arabic translation to the existing English listing. Now when I search for the app in English it pops up as the first option which is perfect, but when I search in Arabic I can't find it at all. I tried and switched the default language to Arabic but nothing changed. I would really appreciate your help and suggestions

I tried searching in Arabic, but I can't find my application. I expect to find it in both languages.

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by HasanAlqaisi** [comment]
Hello, I’m facing the same issue. Have you found a solution?
